css/properties/box-decoration-break.json

```diff
@@ -0,0 +1,78 @@
+{
+  "css": {
+    "properties": {
+      "box-decoration-break": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/box-decoration-break",
+          "support": {
+            "webview_android": {
+              "version_added": true,
+              "notes": "This property is only supported for inline elements."
+            },
+            "chrome": {
+              "prefix": "-webkit-",
+              "version_added": "22",
+              "notes": "This property is only supported for inline elements."
+            },
+            "chrome_android": {
+              "prefix": "-webkit-",
+              "version_added": "4.4",
+              "notes": "This property is only supported for inline elements."
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "32"
```

Comment:
We should have an entry with alternate_name for the -moz-... version.

Reply:
Fixed.